Sources/Extensions/SwiftStdlib/SignedIntegerExtensions.swift

```diff
@@ -78,5 +78,22 @@ public extension SignedInteger {
 		return (self * n).abs / gcd(of: n)
 	}
 	// swiftlint:enable identifier_name
-
+    
+    /// SwifterSwift: Ordinal representation of an integer.
+    ///
+    ///        print((12).ordinalString()) // prints "12th"
+    ///
+    /// - Parameter locale: Locale, default is .current.
+    /// - Returns: String ordinal representation of number in specified locale language. E.g. input 92, output in "en": "92nd"
+    public func ordinalString(locale: Locale = .current) -> String? {
+        if #available(iOS 9.0, macOS 10.11, *) {
```

Comment:
I think wrapping the entire function with `#available` makes more sense here

Reply:
Don't forget to wrap the function with `# if canImport(Foundation)` as well, since `Locale`, `NumberFormatter` and `NSNumber` are not declared in `SwiftStdlib`

Reply:
Is it possible to wrap inside #available all function? I mean, `if #available(iOS 9.0, macOS 10.11, *) {` - it's not the preprocessor macro. It should be inside any function body.

Reply:
@n0an as far as I know it is possible, could you pleas try the following

```swift
    #if canImport(Foundation)
    @available(iOS 9.0, macOS 10.11, *)
    /// SwifterSwift: Ordinal representation of an integer.
    ///
    ///        print((12).ordinalString()) // prints "12th"
    ///
    /// - Parameter locale: Locale, default is .current.
    /// - Returns: String ordinal representation of number in specified locale language. E.g. input 92, output in "en": "92nd"
    public func ordinalString(locale: Locale = .current) -> String? {
        let formatter = NumberFormatter()
        formatter.locale = locale
        formatter.numberStyle = .ordinal
        guard let number = self as? NSNumber else { return nil }
        return formatter.string(from: number)
    }
    #endif
```

Reply:
Yes, it's possible. Done that.
Didn't now about `@available` feature. Thank you!